src/Dto/ActionConfigDto.php

```diff
@@ -109,16 +112,55 @@ public function disableActions(array $actionNames): void
 
     /**
      * @return ActionCollection|array<string,array<string,ActionDto>>
+     *
+     * @deprecated since 4.25.0 and it will be removed in EasyAdmin 5.0.0. Use `getPageActions` or `getActionList` instead.
      */
     public function getActions(): ActionCollection|array
     {
+        trigger_deprecation(
+            'easycorp/easyadmin-bundle',
+            '4.25.0',
+            'Calling "%s" is deprecated and will be removed in 5.0.0. Use `getPageActions` or `getActionList` instead.',
+            __METHOD__,
+        );
+
         return null === $this->pageName ? $this->actions : ActionCollection::new($this->actions[$this->pageName]);
     }
 
+    public function getPageActions(string $pageName): ActionCollection
+    {
+        return ActionCollection::new($this->actions[$pageName]);
+    }
+
+    /**
+     * @return array<string,array<string,ActionDto>>
+     */
+    public function getActionList(): array
```

Comment:
`getAllActions()` instead of `getActionList()`?
```suggestion
    public function getAllActions(): array
```